src/test/kotlin/tech/relaycorp/relaynet/keystores/PrivateKeyStoreTest.kt

```diff
@@ -13,17 +13,16 @@ import tech.relaycorp.relaynet.utils.KeyPairSet
 import tech.relaycorp.relaynet.utils.MockPrivateKeyStore
 import tech.relaycorp.relaynet.utils.PDACertPath
 import tech.relaycorp.relaynet.wrappers.KeyException
-import tech.relaycorp.relaynet.wrappers.x509.CertificateException
+import tech.relaycorp.relaynet.wrappers.privateAddress
 
 @ExperimentalCoroutinesApi
 class PrivateKeyStoreTest {
     private val identityPrivateKey = KeyPairSet.PRIVATE_ENDPOINT.private
-    private val identityCertificate = PDACertPath.PRIVATE_ENDPOINT
 
     private val sessionKeyGeneration = SessionKeyPair.generate()
     private val sessionKeyIdHex = Hex.toHexString(sessionKeyGeneration.sessionKey.keyId)
 
-    private val ownPrivateAddress = identityCertificate.subjectPrivateAddress
+    private val ownPrivateAddress = PDACertPath.PRIVATE_ENDPOINT.subjectPrivateAddress
```

Comment:
Since private key stores will no longer be concerned with certificates, I think it'd be good to replace uses of `PDACertPath.*` with `KeyPairSet.*` to avoid confusion.